OpenSim/Base/OpenSim.cs

```diff
@@ -463,7 +463,7 @@ private void RegisterConsoleCommands()
 
         public override void ShutdownSpecific()
         {
-            if (m_shutdownCommandsFile != String.Empty)
+            if (!String.IsNullOrEmpty(m_shutdownCommandsFile))
```

Comment:
No semantic change: string cannot be null in this case.  The only assignment is in `OpenSim.cs` line 90.